astropy/table/bst.py

```diff
@@ -443,12 +443,10 @@ def _same_prefix(self, val, node, lst):
         return lst
 
     def __str__(self):
```

Comment:
I think you can just remove `__str__` if you want it to be the same as `__repr__`

Reply:
Done